registration/models.py

```diff
@@ -25,6 +26,22 @@
 SHA1_RE = re.compile('^[a-f0-9]{40}$')
 
 
+def get_from_email(site=None):
+    if getattr(settings, 'REGISTRATION_USE_SITE_EMAIL', False):
```

Comment:
We should probably add a docstring here.

Reply:
done